sidecar/dns/server.go

```diff
@@ -214,38 +347,39 @@ func (s *Server) findMatchingServices(question dns.Question, request, response *
 
 }
 
-func validateEndPointTypeTCPAndUDP(value string) (net.IP, error) {
-	ip, _, err := net.SplitHostPort(value)
+func validateEndPointTypeTCPAndUDP(value string) (net.IP,string, error) {
+	ip, port, err := net.SplitHostPort(value)
```

Comment:
`net.SplitHostPort()` doesn't work when no port is specified (e.g. `172.17.0.10`): https://play.golang.org/p/kLnRJxrQAi
I think you'll have to parse it on your own.